exporter/metric/metric.go

```diff
@@ -126,6 +128,16 @@ func (me *metricExporter) Export(ctx context.Context, rm metricdata.ResourceMetr
 	)
 }
 
+// Temporality returns the Temporality to use for an instrument kind.
+func (me *metricExporter) Temporality(ik metric.InstrumentKind) metricdata.Temporality {
+	return metric.DefaultTemporalitySelector(ik)
```

Comment:
How about implementing option like `stdoutmetric`?

- https://github.com/open-telemetry/opentelemetry-go/blob/exporters/stdout/stdoutmetric/v0.34.0/exporters/stdout/stdoutmetric/config.go#L78-L127

Reply:
Our exporter currently assumes we only get cumulative temporality metrics: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/main/exporter/metric/metric.go#L347.  I don't believe Cloud Monitoring doesn't allow delta for custom metrics: https://cloud.google.com/monitoring/api/v3/kinds-and-types#kind-type-combos.  So we should stick with only the default for now.

We can add this in a follow-up if it turns out to be needed.

Reply:
@dashpole I'm totally understand now about Cloud Monitoring temporality metrics spec, make sense.

From your description, I have another concern about use `DefaultTemporalitySelector` might be confusing a bit because not included `CumulativeTemporality` word. But it seems related to otel sdk implementation opinions.

- https://github.com/open-telemetry/opentelemetry-go/blob/sdk/metric/v0.34.0/sdk/metric/reader.go#L115-L117

> We can add this in a follow-up if it turns out to be needed.

IMO, would be adding "note" comment to GoogleCloudPlatform/opentelemetry-operations-go side. But very nits.
Thanks